postgres/init/20-portal.sql

```diff
@@ -638,6 +638,44 @@ CREATE TABLE portal.languages (
 );
 
 
+--
+-- Name: notification_status; Type: TABLE; Schema: portal; Owner: -
+--
+
+CREATE TABLE portal.notification_status (
+    id integer NOT NULL,
+    label character varying(255) NOT NULL
+);
+
+
+--
+-- Name: notification_type; Type: TABLE; Schema: portal; Owner: -
+--
+
+CREATE TABLE portal.notification_type (
+    id integer NOT NULL,
+    label character varying(255) NOT NULL
+);
+
+
+--
+-- Name: notifications; Type: TABLE; Schema: portal; Owner: -
+--
+
+CREATE TABLE portal.notifications (
+    id uuid NOT NULL,
+    company_user_id uuid NOT NULL,
+    date_created timestamp with time zone NOT NULL,
+    title character varying(255) NOT NULL,
```

Comment:
I suggest that we shorten this to actual 150 digits. In many cases pop-up messages are limited to 150-160 characters. Thats in the mobile world the standard as far as I know. Off course we dont have smartphones in focus, but already thinking about being capable to fit into those standards.
I also created a few examples which are currently all running with 60-100 characters.

Reply:
i think thats a good idea. i changed it to 150 digits

Reply:
Merci